Rakefile

```diff
@@ -11,7 +11,10 @@ RuboCop::RakeTask.new
 
 task :default => [:spec, :rubocop, :cucumber]
 
-if ENV['TRAVIS']
+# Skip Ruby 2.4 until coveralls supports simplecov (>= 0.13.0)
+# for Ruby 2.4 compatibility.
+# https://rubygems.org/gems/coveralls
+if ENV['TRAVIS'] && RUBY_VERSION < '2.4'
   ENV['SIMPLECOV']  = 'ci'
```

Comment:
@nodo Yeah, actually it does not matter. This is my favorite. But if the cucumber reviewers does not feel it is problem, I do not change the code.